client/__tests__/QA.test.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import Modal from 'react-modal';
 import { render, screen, cleanup } from '@testing-library/react';
-import QuestionsAndAnswers from '../src/components/QuestionsAndAnswers';
```

Comment:
I notice that some of our test files use the explicit .jsx at the end of their path. I'm not sure if jest accepts both, but we may want to do one or the other for consistency.

Reply:
I'll go back and add '.jsx' on mine.